lib/crystalball/predictor/helpers/affected_examples_detector.rb

```diff
@@ -11,7 +11,13 @@ module AffectedExamplesDetector
         # @return [Array<String>] list of affected examples
         def detect_examples(files, map)
           map.cases.map do |uid, case_map|
-            uid if files.any? { |file| case_map.include?(file) }
+            uid if files.any? do |file|
+              if case_map.is_a?(Array) # support old version of map. TODO: remove it.
```

Comment:
Hm. I don't think we need to support it. How long it will take to generate a new map after you merge this changes? One hour?